src/app/components/window-dialog/window-dialog.component.html

```diff
@@ -5,10 +5,16 @@ <h1 mat-dialog-title>Create a Window</h1>
     <input matInput tabindex="1" [(ngModel)]="name">
   </mat-form-field>
   <p>File:</p>
-  <mat-form-field>
-    <button type="button" mat-button (click)="imgFileInput.click()">Add a photo</button>
-    <input hidden type="file" #imgFileInput (change)="previewImage($event)"/>
-  </mat-form-field>
+
+  <div class = "form-flex-container">
```

Comment:
`class="form-flex-container"` - Weihan [Li]nter

Reply:
i hate you